Events/MarketInformationUpdatedEvent.cs

```diff
@@ -30,9 +32,10 @@ public class MarketInformationUpdatedEvent : Event
 
         /// <summary>The timestamp recorded for this event must be generated from game or server data.
         /// System time (e.g. DateTime.UtcNow) cannot be trusted for reporting to EDDN and may not be used.</summary>
-        public MarketInformationUpdatedEvent(DateTime timestamp, bool inHorizons, string starSystem, string stationName, long? marketId, List<CommodityMarketQuote> commodities, List<string> prohibitedCommodities, List<Module> outfitting, List<Ship> shipyard) : base(timestamp, NAME)
+        public MarketInformationUpdatedEvent(DateTime timestamp, bool inHorizons, bool? allowCobraMkIV, string starSystem, string stationName, long? marketId, List<CommodityMarketQuote> commodities, List<string> prohibitedCommodities, List<Module> outfitting, List<Ship> shipyard) : base(timestamp, NAME)
```

Comment:
Yuck to an optional bool in the middle of a the non-optional args. Make it required bool and change all call sites.

Reply:
Fixed by [`6562958` (#1882)](https://github.com/EDCD/EDDI/pull/1882/commits/656295801cad96ed95067b4851a563ba61b4fd22).